hypershift-operator/controllers/hostedclustersizing/hostedclustersizing_controller_test.go

```diff
@@ -106,9 +106,9 @@ func TestSizingController_Reconcile(t *testing.T) {
 			},
 			nodePoolsForHostedCluster: func(_ context.Context, _ *hypershiftv1beta1.HostedCluster) (*hypershiftv1beta1.NodePoolList, error) {
 				return &hypershiftv1beta1.NodePoolList{Items: []hypershiftv1beta1.NodePool{
-					{Status: hypershiftv1beta1.NodePoolStatus{Replicas: 10}},
-					{Status: hypershiftv1beta1.NodePoolStatus{Replicas: 3}},
```

Comment:
I would also expect to see a new unit test for a no-op when we _want_ to transition but the KAS condition is not set, leading us to not transition